Source/WebCore/workers/DedicatedWorkerThread.cpp

```diff
@@ -55,6 +55,7 @@ Ref<WorkerGlobalScope> DedicatedWorkerThread::createWorkerGlobalScope(const Work
         scope->setActiveServiceWorker(ServiceWorker::getOrCreate(scope.get(), ServiceWorkerData { *params.serviceWorkerData }));
     scope->updateServiceWorkerClientData();
 #endif
+    scope->setWorkerClient(WTFMove(m_workerClient));
```

Comment:
Where `m_workerClient` is created? And why it is not passed to the `DedicatedWorkerGlobalScope::create()` instead of setting it later?

Reply:
It's created in `WorkerMessagingProxy::startWorkerGlobalScope`.